recipe/meta.yaml

```diff
@@ -19,7 +19,7 @@ source:
 
 
 build:
-  number: 1
+  number: 2
   # Since 1.11, power9 seems to be required.
   skip: true  # [ppc64le]
   skip: true  # [cuda_compiler_version == "10.2" or cuda_compiler_version == "11.0" or cuda_compiler_version == "11.1"]
```

Comment:
As conda-forge is on CUDA 11.2+ ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1708 ), these `skip`s are redundant and can be dropped

```suggestion
```